cmd/respawn/respawn.go

```diff
@@ -69,6 +69,12 @@ func run(c *cli.Context) error {
 		panic(err)
 	}
 
+	// Now that we're booted, stop writing debug messages to the console
```

Comment:
`respawn` actually runs in the console so this'll be run again after `ros console switch` in any case. Is there a problem with that? Not sure if calling it more than once will cause an issue.

I'm not sure how a user would override respawn with a `runcmd`. AFAIK respawn can't currently be configured using cloud-config.